test/DefaultRESTClientTest.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, FusionAuth, All Rights Reserved
+ * Copyright (c) 2020, FusionAuth, All Rights Reserved
```

Comment:
We normally preserve the first year, and append to it. `2019-2020` for example.  No need to change necessary, but for future reference.

Reply:
ok, thanks!